Sub-menu-via-event-target/test.js

```diff
@@ -0,0 +1,20 @@
+const ul = document.querySelector(".tree");
+
+const li = document.querySelectorAll("li");
+
+li.forEach((item) => {
+  const span = document.createElement("SPAN");
+  span.classList.add(".p");
```

Comment:
You need to add a class, not a class selector
```suggestion
  span.classList.add("p");
```